src/test/java/ch/wisv/areafiftylan/UserRestIntegrationTest.java

```diff
@@ -1,17 +1,91 @@
 package ch.wisv.areafiftylan;
```

Comment:
This class is very very big, while it also contains tests for the split-up controllers. I would split the tests according to the changes made in #78